src/context.rs

```diff
@@ -2428,7 +2438,7 @@ impl ContextWriter {
   }
 
   pub fn find_mvrefs(&mut self, bo: &BlockOffset, ref_frame: usize,
-                     mv_stack: &mut Vec<CandidateMV>) -> usize {
+                     mv_stack: &mut Vec<CandidateMV>, bsize: BlockSize, is_sec_rect: bool) -> usize {
```

Comment:
Why did you change from using blk.is_sec_rect to needing it as a parameter? Its always set to false now when you call find_mvrefs.

Reply:
blk.is_sec_rect would always be false for now. Same issue as in previous comment: do we pass values as parameters or do we write them into the current blk structure and read them out?